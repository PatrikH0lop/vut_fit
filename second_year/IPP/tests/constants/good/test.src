.ippcode18
DEFVAR TF@a

WRITE int@5
WRITE int@0
WRITE int@-1
WRITE int@5
WRITE int@-2

WRITE string@test
WRITE string@5.6@
WRITE string@__a
WRITE string@string@
WRITE string@**test**
WRITE string@Písanie
WRITE string@αβΓ

WRITE bool@true
WRITE bool@false



