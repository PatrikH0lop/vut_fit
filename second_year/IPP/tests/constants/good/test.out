5
0
-1
5
-2
test
5.6@
__a
string@
**test**
Písanie
αβΓ
true
false
